### ISSUE TITLE
Feat: Can you add it so simply track player turn time across multiple turns until reset?

### ISSUE THREAD

**Issue by digitalpacman** [operation404/boneyard-turn-timer#4]
I am looking to simply use this to figure out which one of my players takes the most time to act, so we can work on it together to get them movin' and groovin'. I would want an option that keeps track of the running total of time all players spend in an encounter on their turn, and, a button to reset the timer.

**Comment by operation404** [comment]
Yeah, I could probably do something like that. I'm not sure where the best place to display that information would be though. I suppose either dumping it to chat or having another entry in the module settings would be the most straightforward.

I'm a bit busy at the moment so I don't know exactly when I'll get to work on this, but I'll make a note for it.

**Comment by valdier1** [comment]
I would think just to the chat. See the defunct PF2e Combat Turn Timers module. A summary at the end of the combat that just shows how long each player took is 100% what I am looking for. I don't actually care about per player timer counting down at all.
